egs/librispeech/ASR/generate-lm.sh

```diff
@@ -0,0 +1,48 @@
+#!/usr/bin/env bash
+
+lang_dir=data/lang_bpe_500
+if [ ! -f $lang_dir/2gram.arpa ]; then
```

Comment:
Could you wrap it in a for loop, like below?
```bash
for ngram in 2 3 5; do
  if [ ! -f $lang_dir/${ngram}gram.arpa ]; then
    ./shared/make_kn_lm.py \
      -ngram-order $ngram \
      -text $lang_dir/transcript_tokens.txt \
      -lm $lang_dir/${ngram}gram.arpa
  fi
  # also for xxx.fst.txt
done

Reply:
Done